### PR TITLE
fix: oracle.RewardBallotWinner detect duplicated uumee

### PR DESCRIPTION
**Pull request by robert-zaremba**
## Description

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] added appropriate labels to the PR
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/umee-network/umee/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items._

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

File: x/oracle/keeper/reward_unit_test.go [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+		out []string
+	}{
+		{[]string{}, []string{"uumee"}},
+		{[]string{"a"}, []string{"uumee", "a"}},
+		{[]string{"x", "a", "heeeyyy"}, []string{"uumee", "x", "a", "heeeyyy"}},
+		{[]string{"x", "a", "uumee"}, []string{"x", "a", "uumee"}},
+	}
+	for i, tc := range tcs {
+		require.Equal(tc.out, voteTargetsWithUmee(tc.in), i)
+	}
```
Comment (toteki) [inline x/oracle/keeper/reward_unit_test.go:22]: comments or named parameters would clarify what this test is doing

**Comment by robert-zaremba** [comment]
@adamewozniak Why are you blocking? This is a clear bug. Should not be blocked!

File: x/oracle/keeper/reward_unit_test.go (hunk 1 above)
Comment (robert-zaremba) [reply x/oracle/keeper/reward_unit_test.go:22]: @toteki isn't this clear? It checks if output of the function `voteTargetWithUmee` with `tc.in` as the input argument equals `tc.out`.

Comment (toteki) [reply x/oracle/keeper/reward_unit_test.go:22]: Maybe a single comment between lines 14-15
```
// Should prepend "uumee" to a slice of denoms, unless it is already present.
```

**Review by toteki — APPROVED** [review]

File: x/oracle/keeper/reward_unit_test.go [hunk 2]
```diff
@@ -0,0 +1,24 @@
+package keeper
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestVoteTargetWithUmee(t *testing.T) {
```
[the remark this answers is not in the document]
Comment (robert-zaremba) [reply x/oracle/keeper/reward_unit_test.go:9]: This is test file ;) We don't crate godoc for test functions.

File: x/oracle/keeper/reward_unit_test.go (hunk 1 above)
Comment (robert-zaremba) [reply x/oracle/keeper/reward_unit_test.go:22]: Ah, will add it.

**Pull request merged**